frontend/javascripts/viewer/controller.tsx

```diff
@@ -309,20 +306,22 @@ class Controller extends React.PureComponent<PropsWithRouter, State> {
 
   render() {
     const status = this.props.controllerStatus;
-    const { user, viewMode } = this.props;
+    const { user, viewMode, isUiReady, isWkInitialized } = this.props;
     const { gotUnhandledError, organizationToSwitchTo } = this.state;
 
-    if (status === "loading") {
-      return <BrainSpinner />;
+    let cover = null;
+    // Show the brain spinner during loading and until the UI is ready
+    if (status === "loading" || (status === "loaded" && !isUiReady)) {
+      cover = <BrainSpinner />;
     } else if (status === "failedLoading" && user != null) {
-      return (
+      cover = (
         <BrainSpinnerWithError
           gotUnhandledError={gotUnhandledError}
           organizationToSwitchTo={organizationToSwitchTo}
         />
       );
     } else if (status === "failedLoading") {
-      return (
```

Comment:
Here is the cause for bug noticed here: https://scm.slack.com/archives/C5AKLAV0B/p1761766742395699.

The issue seems to be that the status is still loading but wk is already initialized. In that case the previous logic just rendered the brain spinner. The new code now also tries to render the `PlaneController` when `status === "loading" && isWkInitialized === true`. And `PlaneController` tries to access the scene controller which is not yet re-initialized.

Reverting this code seems to fix the bug. But not sure whether this is actually hiding a bug 🤔